support/certs/tls.go

```diff
@@ -30,9 +30,11 @@ const (
 	ValidityOneYear  = 365 * ValidityOneDay
 	ValidityTenYears = 10 * ValidityOneYear
 
-	CAHashAnnotation   = "hypershiftlite.openshift.io/ca-hash"
-	CASignerCertMapKey = "ca.crt"
-	CASignerKeyMapKey  = "ca.key"
+	CAHashAnnotation    = "hypershiftlite.openshift.io/ca-hash"
+	CASignerCertMapKey  = "ca.crt"
+	CASignerKeyMapKey   = "ca.key"
+	TLSSignerCertMapKey = "tls.crt"
```

Comment:
can we add a comment here documenting why/how CASignerCertMapKey and CASignerKeyMapKey are different from TLSSignerCertMapKey and TLSSignerKeyMapKey?
And clarifying that the latter is programatically enforced to have the same content than the former?

Reply:
Added